cpp/src/cylon/join/join.cpp

```diff
@@ -257,8 +280,8 @@ arrow::Status do_inplace_sorted_join(const std::shared_ptr<arrow::Table> &left_t
 template<typename ARROW_T, typename CPP_KEY_TYPE>
 static inline arrow::Status do_sorted_join(const std::shared_ptr<arrow::Table> &left_tab,
                                            const std::shared_ptr<arrow::Table> &right_tab,
-                                           int64_t left_join_column_idx,
-                                           int64_t right_join_column_idx,
+                                           std::vector<int64_t> left_join_column_idxs,
```

Comment:
let's use `const vector<int32_t> &`s here

Reply:
I guess this is also there in the missing two commits .